packages/server/lib/errors.coffee

```diff
@@ -944,6 +944,16 @@ getMsgByType = (type, arg1 = {}, arg2, arg3) ->
 
       To avoid this error, ensure that there are no other instances of Firefox launched by Cypress running.
       """
+    when "FOLDER_NOT_ACCESSIBLE"
```

Comment:
i find it a little confusing: error is not accessible, but the real error is not writable